acceptance/cmd_server_test.go

```diff
@@ -15,7 +15,8 @@ import (
 func StartCmdServer(t *testing.T) *testserver.Server {
 	server := testserver.New(t)
 
-	server.Handle("/", func(r *http.Request) (any, int) {
+	// {$} is a wildcard that only matches the end of the URL.
+	server.Handle("/{$}", func(r *http.Request) (any, int) {
```

Comment:
If this is needed to disambiguate from the default handler, please mention it in the comment.

Reply:
Added a comment